controlsfx/src/main/java/org/controlsfx/validation/ValidationSupport.java

```diff
@@ -158,6 +158,24 @@ public void redecorate() {
             });
         }
     }
+
+    /**
+     * Triggers validation for all known components.
+     * It is only necessary to call this if it is needed to revalidate even if the value of the control has not changed.
+     */
+    public void revalidate() {
+        for (Control target : getRegisteredControls()) {
+            target.fireEvent(new ValidateEvent());

```

Comment:
I wonder why ValidationEvent instance has to be created every time? 
It does not cary any specific information. I'd think singleton instance should be enough.

Reply:
Event objects are not meant to be reused. If you look at the parent `javafx.event.Event` class it does carry state such as the `consumed` field. Logically also, it is a different occurrence of the event, so it should be a different object.

Reply:
Not a big deal... was simply wondering if optimization is possible.